openquake/qa_tests_data/classical/case_1/expected/hazard_curve-smltp_b1-gsimltp_b1.csv

```diff
@@ -1,3 +1,3 @@
 # source_model_tree_path=('b1',),gsim_tree_path=('b1',),investigation_time=1.0
-lon,lat,PGA-0.1,PGA-0.4,PGA-0.6,SA(0.1)-0.1,SA(0.1)-0.4,SA(0.1)-0.6
-0.00000,0.00000,4.570135E-01,5.862679E-02,6.866165E-03,6.086748E-01,3.308305E-01,2.014712E-01
+lon,lat,depth,PGA-0.1,PGA-0.4,PGA-0.6,SA(0.1)-0.1,SA(0.1)-0.4,SA(0.1)-0.6
+0.00000,0.00000,-0.10000,4.528811E-01,5.589804E-02,5.585141E-03,6.074420E-01,3.254916E-01,1.964219E-01
```

Comment:
I checked these values just using the hazardlib, as we were discussing, and get the same result